test/jdk/java/lang/management/ThreadMXBean/VirtualThreadDeadlocks.java

```diff
@@ -98,6 +97,9 @@ public static void main(String[] args) throws Exception {
     private static void awaitBlocked(Thread thread) throws InterruptedException {
         while (thread.getState() != Thread.State.BLOCKED) {
             Thread.sleep(10);
+            if (thread.getState() == Thread.State.TERMINATED) {
```

Comment:
Thread.State.TERMINATED == thread.getState(). Does it make sense?

Reply:
Not sure. What is the goal?

Reply:
I assume this test is catch the catch where the thread terminates without blocking. Using isAlive might be cleaner for people to understand.